htpasswd/auth_test.go

```diff
@@ -3,6 +3,7 @@ package htpasswd
 import (
```

Comment:
There is no test checking that the authentication works for users in different files.